app/config/collections.php

```diff
@@ -3580,6 +3580,139 @@
+            ],
+            [
+                '$id' => ID::custom('resources'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => [],
+                'array' => true,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('statusCounters'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 3000,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('resourceData'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 131070,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('errorData'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 65535,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => ['json'],
+            ],
+            [
+                '$id' => ID::custom('search'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ]
+        ],
+        'indexes' => [
+            [
+                '$id' => '_key_status',
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['status'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => '_key_stage',
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['stage'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => '_key_source',
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['source'],
+                'lengths' => [Database::LENGTH_KEY],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => '_key_statusCounters',
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['statusCounters'],
+                'lengths' => [Database::LENGTH_KEY],
```

Comment:
statusCounters attribute size is 3000, Only 255 chars (Database::LENGTH_KEY) will be indexes.
Please double-check if it is enough for your queries, can be up to 768 MAX.

Reply:
We don't perform any queries on this attribute, we may be best off just not having a index for it